### DISCUSSION TITLE
[V2] cargo test STATUS_ENTRYPOINT_NOT_FOUND error.

### DISCUSSION THREAD

**Discussion by probaku1234** [tauri-apps/tauri#11179] [Q&A] [answered]
I'm trying to implement unit testing for tauri command function. 
When I run this test, It fails with the STATUS_ENTRYPOINT_NOT_FOUND error.
```
    use tauri::test::{mock_builder};

    fn create_app<R: tauri::Runtime>(builder: tauri::Builder<R>) -> tauri::App<R> {
        builder
            // .invoke_handler(tauri::generate_handler![ping])
            // remove the string argument to use your app's config file
            .build(tauri::generate_context!("tauri.conf.json"))
            .expect("failed to build app")
    }
    

    #[test]
    fn test_greet() {
        let app = create_app(mock_builder());
    }
```

The original code is from the example code of the [link](https://docs.rs/tauri/2.0.0-rc.16/tauri/test/index.html).
I commented out most of the code to find which code causes the error, and It seems `builder: tauri::Builder<R>` is the one.
Any idea how to fix this issue?

**Comment by amrbashir** [comment, marked as answer]
We have a workaround for our repo here https://github.com/tauri-apps/tauri/blob/60a5aea53db02ae6af325812ab97555f2c013d70/examples/api/src-tauri/build.rs#L24-L51 and https://github.com/tauri-apps/tauri/blob/60a5aea53db02ae6af325812ab97555f2c013d70/.cargo/config.toml#L2-L4 and https://github.com/tauri-apps/tauri/blob/60a5aea53db02ae6af325812ab97555f2c013d70/crates/tauri-build/src/windows-app-manifest.xml#L1-L14

I tried to fix this in the tauri crate itself but didn't find a way to, unfortunately.

**Reply by savageKarl** [reply]
i got same issue.

i already set 
```

[env]
__TAURI_WORKSPACE__ = "true"
```

but isn't working.

i still got `(exit code: 0xc0000139, STATUS_ENTRYPOINT_NOT_FOUND)`

win10  OS.

please,help me,thank you so much;

@amrbashir

**Comment by arashi-dev** [comment]
I am still getting this issue!
